website/src/components/OrganismTableSelector.tsx

```diff
@@ -0,0 +1,139 @@
+        }
+        setExpandedHeaders(updatedExpandedHeaders);
+    };
+
+    useEffect(() => {
+        if (selectedOrganism) {
+            setGroupedFields(groupFieldsByHeader(selectedOrganism.inputFields, selectedOrganism.metadata));
+        }
+    }, [selectedOrganism]);
+
+    return (
+        <div>
+            <div>
+                <select id='organism-select' onChange={handleOrganismSelect} className='border border-gray-300 p-2'>
+                    <option value=''>-- Select an Organism --</option>
+                    {organisms.map((organism) => (
+                        <option key={organism.key} value={organism.key}>
+                            {organism.displayName}
+                        </option>
+                    ))}
+                </select>
+            </div>
+
+            {selectedOrganism && (
+                <div className='mt-6'>
+                    <h1 className='text-2xl font-bold mb-4'>{selectedOrganism.displayName}</h1>
+                    <div>
+                        You can download all metadata fields and their descriptions here:{' '}
+                        <a
+                            href={routes.metadataOverview(selectedOrganism.key)}
+                            className='text-primary-700  opacity-90'
+                        >
+                            metadata_fields_descriptions.csv
+                        </a>
+                    </div>
+                    {Array.from(groupedFields.entries()).map(([header, fields]) => (
+                        <div key={header} className='mb-8'>
+                            <h3
+                                className='text-lg font-semibold mb-4 cursor-pointer'
+                                onClick={() => toggleHeader(header)}
+                            >
+                                {header}
+                                <IwwaArrowDown className='inline-block -mt-1 ml-1 h-4 w-4' />
+                            </h3>
+                            <div
+                                className={`transition-all duration-300 ${
+                                    expandedHeaders.has(header) ? 'block' : 'sr-only'
+                                }`}
+                                data-table-header={header}
+                            >
+                                <MetadataTable fields={fields} metadata={selectedOrganism.metadata} />
+                            </div>
+                        </div>
+                    ))}
+                </div>
+            )}
+        </div>
+    );
+};
+
+export default OrganismTableSelector;
+
+type TableProps = {
+    fields: InputField[];
+    metadata: Metadata[];
+};
+
+const MetadataTable: FC<TableProps> = ({ fields, metadata }) => {
+    return (
+        <table className='table-auto border-collapse border border-gray-200 w-full'>
+            <thead>
+                <tr>
+                    <th className='border border-gray-300 px-4 py-2' style={{ width: '20%' }}>
```

Comment:
```suggestion
                    <th className='border border-gray-300 px-4 py-2 w-[20%]'>
```

(In this case you could actually use `w-1/5`, but for the other special cases you can't)